docker-compose.yml

```diff
@@ -32,8 +32,6 @@ services:
     container_name: node-api
     ports:
       - 3000:3000
```

Comment:
why are we removing the volumes here? shouldn't we use the new WORKDIR? `.:/api`

Reply:
We can discuss this later today at our follow-up meeting.